org.abapgit.adt.ui/src/org/abapgit/adt/ui/internal/repositories/RepositoryModifiedObjects.java

```diff
@@ -0,0 +1,61 @@
+package org.abapgit.adt.ui.internal.repositories;
+
+import java.util.List;
+
+import org.abapgit.adt.backend.model.agitpullmodifiedobjects.IAbapGitObject;
+
+/**
+ * Implements IRepositoryModifiedObjects
+ *
+ * @author I517012
+ */
+public class RepositoryModifiedObjects implements IRepositoryModifiedObjects {
+	@Override
+	public int hashCode() {
+		final int prime = 31;
+		int result = 1;
+		result = prime * result + ((this.repositoryURL == null) ? 0 : this.repositoryURL.hashCode());
+		return result;
+	}
+
+	@Override
+	public boolean equals(Object obj) {
+		if (this == obj) {
+			return true;
+		}
+		if (obj == null) {
+			return false;
+		}
+		if (getClass() != obj.getClass()) {
+			return false;
+		}
+		RepositoryModifiedObjects other = (RepositoryModifiedObjects) obj;
```

Comment:
should we do a check if( obj instanceof RepositoryModifiedObjects ) ? to prevent any wrong conversions

Reply:
I don't think we need that check. Line 29 and 30 take care of it.

Reply:
ok